lang/python/python-et_xmlfile/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-et_xmlfile
+PKG_VERSION:=1.0.1
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+
+PKG_SOURCE:=et_xmlfile-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/e/et_xmlfile
+PKG_HASH:=614d9722d572f6246302c4491846d2c393c199cfa4edc9af593437691683335b
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-et_xmlfile-$(PKG_VERSION)
+PKG_MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
+
+include $(INCLUDE_DIR)/package.mk
+include ../python-package.mk
+include ../python3-package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-et_xmlfile/Default
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Low memory library for creating large XML files.
+  URL:=https://bitbucket.org/openpyxl/et_xmlfile
+endef
+
+define Package/python-et_xmlfile
+$(call Package/python-et_xmlfile/Default)
+  DEPENDS:= \
+	+PACKAGE_python-et_xmlfile:python-light \
```

Comment:
doesn't this ( `+PACKAGE_python-et_xmlfile` ) cause circular deps  with seafile ?

Reply:
It apparently works.  I am getting circular deps with current master (before applying this PR), caused by `DEPENDS:=@!arc` in libseccomp.  If I comment that out, both master and my branch have no circular deps.

Reply:
ack